components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -0,0 +1,146 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomDrawerPresentationController.h"
+
+#import "MDCBottomDrawerViewController.h"
+#import "private/MDCBottomDrawerContainerViewController.h"
+
+static UIColor *MDCBottomDrawerOverlayBackgroundColor(void) {
+  return [UIColor colorWithWhite:0 alpha:0.4f];
+}
+
+@interface MDCBottomDrawerPresentationController () <UIGestureRecognizerDelegate>
+
+/**
+ A semi-transparent view that darkens the visible main view when the drawer is displayed.
```

Comment:
Suggestion: I believe this is referred to as [a "scrim" in the Material guidelines](https://material.io/design/environment/surfaces.html#attributes). Can we make use of that terminology so make the purpose clear as well as its behavior?

Reply:
thanks, updated